packages/components/src/internal/components/ontology/ConceptInformationTabs.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {FC, memo, ReactNode, SyntheticEvent, useCallback} from "react";
+import {Col, Nav, NavItem, Panel, Row, Tab, TabContainer} from "react-bootstrap";
+import {ConceptModel} from "./models";
+
+export interface ConceptInformationTabsProps {
+    concept?: ConceptModel;
+    activeTab?: ConceptInfoTabs;
+    onTabChange?: (string) => void;
+    onPathChange?: () => void;
+}
+
+export const enum ConceptInfoTabs {
+    CONCEPT_OVERVIEW_TAB = 'overview',
+    PATH_INFO_TAB = 'pathinfo',
+}
+
+export const ConceptInformationTabs: FC<ConceptInformationTabsProps> = memo(props => {
+    const { concept, activeTab, onTabChange, onPathChange } = props;
+
+    const onSelectionChange = useCallback(
+        (event: SyntheticEvent<TabContainer, Event>) => {
+            const tab: ConceptInfoTabs = event as any; // Crummy cast to make TS happy
+            onTabChange(tab);
+        },
+        [onTabChange]
+    );
+
+    return (
+        <>
+            <div>
+                <Tab.Container
+                    id="concept-information-tabs"
+                    onSelect={onSelectionChange}
+                    activeKey={activeTab}
+                    defaultActiveKey={ConceptInfoTabs.CONCEPT_OVERVIEW_TAB}
+                >
+                    <Row className="clearfix">
+                        <Col>
+                            <Nav bsStyle="tabs">
+                                <NavItem eventKey={ConceptInfoTabs.CONCEPT_OVERVIEW_TAB}>Overview</NavItem>
+                                <NavItem eventKey={ConceptInfoTabs.PATH_INFO_TAB}>Path Information</NavItem>
+                            </Nav>
+                        </Col>
+                        <Col>
+                            <Tab.Content animation>
+                                <Tab.Pane
+                                    className="ontology-concept-overview-container"
+                                    eventKey={ConceptInfoTabs.CONCEPT_OVERVIEW_TAB}
+                                >
+                                    <ConceptOverviewPanel {...concept} />
```

Comment:
Really just a syntax preference for me.

This enabled using the ConceptModel object as a the props object for the ConceptOverviewPanel, e.g. 
FC<ConceptModel> vs. FC<{ concept: ConceptModel }>. Then within the panel using the deconstructed fields rather than dereferencing.